### PR TITLE
Rename import

### PR DESCRIPTION
**Pull request by caro3801**
Problem: The IDE shows the following errors
`(Cannot find module '@lib/i18n' or its corresponding type declarations.ts(2307))`

Proposal:
Update the path from `@lib `to `@/`
Changes are mainly due to update in [vite.config.js](https://github.com/ICIJ/murmur/pull/68/commits/1268b8ce40534072c0fe0279ea5a5e9b8efbcadd#diff-58e6f63d87181b1c6a8cb6e5f1691df04aa32854456efcd52ca71c8541375d26)

### CODE REVIEW

File: docs/components/DocsMenuSection.vue [hunk 1]
```diff
@@ -26,8 +26,8 @@
   import { faAngleUp } from '@fortawesome/free-solid-svg-icons/faAngleUp'
   import { faBook } from '@fortawesome/free-solid-svg-icons/faBook'
 
-  import SlideUpDown from '@lib/components/SlideUpDown.vue'
-  import { library, default as Fa } from '@lib/components/Fa'
+  import SlideUpDown from '@/components/SlideUpDown.vue'
```
Comment (marieglr) [inline docs/components/DocsMenuSection.vue:29]: Not for this PR but now that we have ESlint we should have a rule for import extensions :)

File: lib/components/AccordionStep.vue [hunk 2]
```diff
@@ -61,8 +61,8 @@
 
 <script lang="ts">
 import Vue, { VueConstructor} from "vue"
-import {AccordionKey} from "@lib/keys"
-import {Accordion, Step} from "@lib/types";
+import {AccordionKey} from "@/keys"
```
Comment (marieglr) [inline lib/components/AccordionStep.vue:64]: Another linting issue: spaces inside brackets is not consistent :thinking:

File: docs/components/App.vue [hunk 3]
```diff
@@ -28,7 +28,7 @@
 
 <script>
 import { faBars } from '@fortawesome/free-solid-svg-icons/faBars'
-import { library, default as Fa } from '@lib/components/Fa'
+import { library } from '@/components/Fa'
```
Comment (marieglr) [inline docs/components/App.vue:31]: nice catch!

**Review by marieglr — APPROVED** [review]
Feels good :broom:

File: docs/components/DocsMenuSection.vue (hunk 1 above)
Comment (caro3801) [reply docs/components/DocsMenuSection.vue:29]: Definitely :+1:

**Pull request merged**